src/guides/v2.3/graphql/queries/products.md

```diff
@@ -2,7 +2,7 @@
 group: graphql
 title: products query
 redirect_from:
-  - /guides/v2.3/graphql/reference/products.html
+  - graphql/reference/products.html
```

Comment:
I don't understand why you've made the path shorter here, but longer in `create-graphqls-file.md`. The original looks correct.

Reply:
Fixed